src/Metrics.h

```diff
@@ -278,9 +295,12 @@ class TopN final : public Metric
     {
         auto section = json::array();
         auto items = _fi.get_frequent_items(datasketches::frequent_items_error_type::NO_FALSE_NEGATIVES);
+        auto threshold = _get_threshold();
         for (uint64_t i = 0; i < std::min(_top_count, items.size()); i++) {
-            section[i]["name"] = formatter(items[i].get_item());
-            section[i]["estimate"] = items[i].get_estimate();
+            if (items[i].get_estimate() >= threshold) {
+                section[i]["name"] = formatter(items[i].get_item());
+                section[i]["estimate"] = items[i].get_estimate();
+            }
```

Comment:
since this is sorted, we can add an `else` here to break when estimate is < threshold

Reply:
done